### PR TITLE
ODQA- 3155 code review(odd or even)

### PR DESCRIPTION
**Pull request by perivale**
ODQA-odd or even number

### CODE REVIEW

File: src/Shilpa.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class Shilpa {
+    public static void main(String[] args) {
+        int i=2;
+        if(i%2==0){
+            System.out.println("this is even number");
+        }
+    }
+
+
```
Comment (MeghaJadvani) [inline src/Shilpa.java:9]: remove extra space

Comment (savkin88) [reply src/Shilpa.java:9]: u are right

Comment (perivale) [reply src/Shilpa.java]: Done

**Comment by Jalpa19** [comment]
can you see my comment

**Review by MeghaJadvani — APPROVED** [review]

**Review by Jalpa19 — APPROVED** [review]

**Pull request merged**